packages/react-menu/src/components/Menu/Menu.types.ts

```diff
@@ -119,7 +119,12 @@ export interface MenuState extends MenuProps {
 /**
  * Data attached to open/close events
  */
-export interface OnOpenChangeData extends Pick<MenuState, 'open'> {}
+export interface OnOpenChangeData extends Pick<MenuState, 'open'> {
```

Comment:
```suggestion
export interface MenuOnOpenChangeData extends Pick<MenuState, 'open'> {
```

(nit) this probably should include component's name.